lib/hawkular/alerts/alerts_api.rb

```diff
@@ -523,8 +523,24 @@ def to_h
   # server and not 'rubyfied'. So 'alertId' and not 'alert_id'
   # Check http://www.hawkular.org/docs/rest/rest-alerts.html#Alert for details
   class Alert < OpenStruct
+    attr_accessor :lifecycle
+
     def initialize(alert_hash)
       super(alert_hash)
+      @lifecycle = alert_hash['lifecycle']
+    end
+
+    def ackBy
+      statusBy('ACKNOWLEDGED')
+    end
+
+    def resolvedBy
+      statusBy('RESOLVED')
+    end
+
+    def statusBy(status)
+      a = @lifecycle.nil? ? [] : @lifecycle.select { |l| l['status'].eql? status }
+      a.empty? ? '' : a.last['user']
```

Comment:
that's a good question, let me verify that '' is correct.  If not I'll change to nil.